File: Habitica/src/main/java/com/habitrpg/android/habitica/data/FAQRepository.java
package com.habitrpg.android.habitica.data;

import com.habitrpg.android.habitica.models.FAQArticle;

import io.reactivex.Flowable;
import io.realm.RealmResults;

public interface FAQRepository extends BaseRepository {
    Flowable<RealmResults<FAQArticle>> getArticles();

}

File: Habitica/src/main/java/com/habitrpg/android/habitica/data/FAQRepository.kt
package com.habitrpg.android.habitica.data

import com.habitrpg.android.habitica.models.FAQArticle
import kotlinx.coroutines.flow.Flow

interface FAQRepository : BaseRepository {
    fun getArticles(): Flow<List<FAQArticle>>

    fun getArticle(position: Int): Flow<FAQArticle>
}
